src/uikit/Select.js

```diff
@@ -3,91 +3,179 @@ import Downshift from 'downshift';
 
 import downChevronIcon from '../assets/icon-chevron-down-grey.svg';
 
+export const OptionDropdownWrapperCss = `
+  position: absolute;
+  background: white;
+  min-width: 100%;
+  z-index: 1;
+  border: 1px solid rgba(0, 0, 0, 0.05);
+  box-sizing: border-box;
+  cursor: pointer;
+  padding: 5px;
+  top: 100%;
+`;
+
+const DisabledDropdownOption = `
+  color: lightgrey
+`;
+
+export const DropDownOption = ({
+  item,
+  itemClassName,
+  DisabledDropdownOption,
+  isItemDisabled,
+  onToggle,
+  selectItem,
+  onDisabledItemClick,
+  getItemProps,
+}) => (
+  <div
+    {...getItemProps({ item })}
+    key={item}
+    css={`
+      cursor: pointer;
+      padding: 5px;
+      ${itemClassName};
+      ${isItemDisabled({ item }) ? DisabledDropdownOption : ''};
```

Comment:
maybe there was some code change that led to this pattern being antiquated, but I'm not sure the whole passing `DisabledDropdownOption` through props thing is necessary.
```
  ${isItemDisabled({ item })} ? `color: lightgrey;` : ``}
```